qa/vagrant/src/test/resources/packaging/scripts/plugin_test_cases.bash

```diff
@@ -197,6 +197,15 @@ fi
     assert_file_exist "$ESHOME/plugins/site-example/_site/index.html"
 }
 
+@test "[$GROUP] check the installed plugins can be listed with 'plugins list' and result matches the list of plugins in plugins pom" {
+    list_plugins | tail -n +2 | sed 's/^......//' | \
```

Comment:
I don't think I'd pull `list_plugins` into it's own thing. Mostly because its just used once and the name confused me at first. I thought it was a thing from the cat api and didn't even realized that `bin/pluin list` existed.